Source/WebCore/Modules/highlight/Highlight.cpp

```diff
@@ -103,5 +103,11 @@ void Highlight::repaint()
         repaintRange(data->range());
 }
 
+void Highlight::setPriority(int priority)
+{
+    if (priority >= 0)
```

Comment:
maybe this can be an `ASSERT` instead of / in addition to this check?

Reply:
Yeah, this shouldn't be silent I think

Reply:
would it look something like this `ASSERT(!priority);`?

Reply:
It would be ASSERT(priority >= 0). But you should probably address Aditya’s comment about if negative priorities are actually allowed.